src/accordion/accordion.css

```diff
@@ -0,0 +1,5 @@
+.container {
+  border: 1px solid LightGrey;
+  border-radius: 3px;
+  box-shadow: 0 1px 2px LightGrey;
```

Comment:
Heh, probably don't want to have drop shadow on here that someone will have to take off. Was following the WAI-ARIA examples styling.